code/modules/cargo/exports/lavaland.dm

```diff
@@ -4,52 +4,56 @@
 /datum/export/lavaland/minor
 	cost = CARGO_CRATE_VALUE * 20
 	unit_name = "minor lava planet artifact"
-	export_types = list(/obj/item/immortality_talisman,
-						/obj/item/book_of_babel,
-						/obj/item/wisp_lantern,
-						/obj/item/katana/cursed,
-						/obj/item/clothing/glasses/godeye,
-						/obj/item/melee/ghost_sword,
-						/obj/item/clothing/neck/necklace/memento_mori,
-						/obj/item/organ/internal/heart/cursed/wizard,
-						/obj/item/clothing/suit/hooded/cloak/drake,
-						/obj/item/ship_in_a_bottle,
-						/obj/item/clothing/shoes/clown_shoes/banana_shoes,
-						/obj/item/gun/magic/staff/honk,
-						/obj/item/knife/envy,
-						/obj/item/gun/ballistic/revolver/russian/soul,
-						/obj/item/veilrender/vealrender,
-						/obj/item/clothing/suit/hooded/berserker,
-						/obj/item/freeze_cube,
-						/obj/item/soulstone/anybody/mining,
-						/obj/item/clothing/gloves/gauntlets,
-						/obj/item/jacobs_ladder,
-						)
+	export_types = list(
+		/obj/item/immortality_talisman,
+		/obj/item/book_of_babel,
+		/obj/item/wisp_lantern,
+		/obj/item/organ/internal/cyberimp/arm/shard/katana,
+		/obj/item/clothing/glasses/godeye,
+		/obj/item/clothing/neck/necklace/memento_mori,
+		/obj/item/organ/internal/heart/cursed/wizard,
+		/obj/item/clothing/suit/hooded/cloak/drake,
+		/obj/item/ship_in_a_bottle,
+		/obj/item/clothing/shoes/clown_shoes/banana_shoes,
+		/obj/item/gun/magic/staff/honk,
+		/obj/item/knife/envy,
+		/obj/item/gun/ballistic/revolver/russian/soul,
+		/obj/item/veilrender/vealrender,
+		/obj/item/clothing/suit/hooded/berserker,
+		/obj/item/freeze_cube,
+		/obj/item/soulstone/anybody/mining,
+		/obj/item/clothing/gloves/gauntlets,
+		/obj/item/jacobs_ladder,
+		/obj/item/borg/upgrade/modkit/lifesteal,
+	)
 
 /datum/export/lavaland/major //valuable chest/ruin loot, minor megafauna loot
 	cost = CARGO_CRATE_VALUE * 40
 	unit_name = "lava planet artifact"
-	export_types = list(/obj/item/guardiancreator/miner,
-						/obj/item/rod_of_asclepius,
-						/obj/item/dragons_blood,
-						/obj/item/lava_staff,
-						/obj/item/prisoncube,
-						)
+	export_types = list(
+		/obj/item/guardiancreator/miner,
+		/obj/item/rod_of_asclepius,
+		/obj/item/dragons_blood,
+		/obj/item/melee/ghost_sword,
```

Comment:
Are we sure the ghost sword should be worth this much?

Reply:
This is where drake loot appears to be kept. For consistency, I don't see why not.